src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -227,6 +227,7 @@ public abstract partial class JsonNamingPolicy
     {
         protected JsonNamingPolicy() { }
         public static System.Text.Json.JsonNamingPolicy CamelCase { get { throw null; } }
+        public static System.Text.Json.JsonNamingPolicy SnakeCase { get { throw null; } }
         public abstract string ConvertName(string name);
```

Comment:
/// <summary>
        /// Property names are not changed
        /// </summary>
`public static System.Text.Json.JsonNamingPolicy Deafult{ get { throw null; } }`